src/main/java/views/TeamEditorPane.java

```diff
@@ -0,0 +1,241 @@
+package views;
+
+import controllers.SceneManager;
+import javafx.collections.FXCollections;
+import javafx.geometry.Pos;
+import javafx.scene.control.Button;
+import javafx.scene.control.ComboBox;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+
+import java.util.*;
+
+import javafx.scene.text.Text;
+import models.*;
+
+
+public class TeamEditorPane extends GamePane {
+
```

Comment:
Same issue as in LeagueSimulatorPane, lots of repeated code and member variables labeled 1,2,3...

Reply:
+1

Reply:
Agreed

Reply:
+3